alpenhorn/auto_import.py

```diff
@@ -346,29 +341,13 @@ def get_miscfile_data(path, _file):
             if "finish_time" in metadata:
                 start_time = metadata["finish_time"]
         except KeyError:
-            pass
-
-        # Generate a list of all regular files
-        file_list = [
-            {
-                "file": _file,
-                "name": member.name,
-                "size_b": member.size,
-                "mtime": member.mtime,
-            }
-            for member in f
-            if member.isfile()
-        ]
-
-    return (
-        {
-            "file": _file,
-            "start_time": start_time,
-            "finish_time": finish_time,
-            "data_type": data_type,
-        },
-        file_list,
-    )
+            metadata = None
+    return {
+        "start_time": start_time,
+        "finish_time": finish_time,
+        "data_type": data_type,
+        "metadata": metadata,
```

Comment:
Do you want to remove the `start_time` and `finish_time` entries from `metadata`? Then the same information doesn't appear in two places.

Reply:
I've left them in if there's an error parsing the time.